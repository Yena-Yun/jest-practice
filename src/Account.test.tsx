import Account from './Account';
import { render } from '@testing-library/react';

it('matches snapshot', () => {
	const utils = render(<Account name="셰릴아리" mbti="ISTP" />);
	expect(utils.container).toMatchSnapshot();
})

it('shows the props correctly', () => {
	const utils = render(<Account name="셰릴아리" mbti="ISTP" />);
	utils.getByText('셰릴아리');
	utils.getByText('ISTP');
})