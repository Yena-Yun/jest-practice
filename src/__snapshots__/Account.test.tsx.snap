// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <h2>
      셰릴아리
    </h2>
    <p>
      ISTP
    </p>
  </div>
</div>
`;
